setenv.sh

```diff
@@ -24,4 +26,4 @@ export TORQPROCESSES=${KDBAPPCONFIG}/process.csv
 
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$KDBLIB/l32
 
-export TORQMONIT=${TORQHOME}/logs/monit                                                             # set the folder for monit outputs  
+export TORQMONIT=${TORQHOME}/logs/monit                                                             # set the folder for monit outputs
```

Comment:
This is currently unused and actually refers to a different location to the log location defined in `monit/bin/monit.sh` (which is `monit/logs`).